test_support/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+use libcnb_test::{
+    assert_contains, BuildConfig, BuildpackReference, ContainerConfig, TestContext, TestRunner,
+};
+use std::time::Duration;
+
+const PORT: u16 = 8080;
+
+pub enum Builder {
+    Heroku20,
+    Heroku22,
+}
+
+pub fn get_function_invoker_buildpacks() -> Vec<BuildpackReference> {
+    vec![
+        BuildpackReference::Other(String::from("heroku/nodejs-engine")),
+        BuildpackReference::Other(String::from("heroku/nodejs-npm")),
+        BuildpackReference::Crate,
+    ]
+}
+
+pub fn get_builder_name(builder: Builder) -> &'static str {
```

Comment:
Maybe it might make sense to implement the [`Display` trait](https://doc.rust-lang.org/std/fmt/trait.Display.html) instead here? That'd give us the ability to use `builder.to_string()` or `println!("Builder: {builder}")` with this type.